src/drive/web/modules/queries.js

```diff
@@ -343,7 +343,16 @@ export const prepareSuggestionQuery = () => ({
       $or: [{ $exists: false }, { $eq: false }]
     }
   },
-  fields: ['_id', 'trashed', 'dir_id', 'name', 'path', 'type', 'mime'],
+  fields: [
+    '_id',
+    'trashed',
+    'dir_id',
+    'name',
+    'path',
+    'type',
+    'mime',
+    'metadata'
```

Comment:
Une chose est sûre, c'est qu'on peut pas garder `metadata` au complet. Soit on arrive à aller lire quelques champs particuliers, soit on change la façon de checker si c'est une note. Car `metadata` sur les notes, ça peut vite peser des centaines de ko par note.

Reply:
j'ai recréé un helper isNote, car les metadata peuvent être volumineuses.

Reply:
I don't know if I like this idea. 

You can add to the select(['metadata.title', 'metadata.version']) . title + version are not big. So the helper can be if(metata.title && metadata.version...) 

See a test query with metadata.datetime to see the result. 

<img width="923" alt="Capture d’écran 2022-08-22 à 11 31 33" src="https://user-images.githubusercontent.com/1107936/185889178-59411489-e10d-4f13-b067-429d0da778c1.png">

What do you think?

Reply:
I have only one note in my local Drive.
When adding:
```
    'metadata.content',
    'metadata.schema',
    'metadata.title',
    'metadata.version'
```
as fields, the size of request increase 
from 650ko / 39,87 ko transferred in 538ms 
to 657,97 ko / 41,61ko transferred in 578ms.

![Capture d’écran 2022-08-22 à 12 03 37](https://user-images.githubusercontent.com/8363334/185896628-463967d9-3029-4fbf-ae7b-462123e19cd1.png)
![Capture d’écran 2022-08-22 à 12 05 08](https://user-images.githubusercontent.com/8363334/185896635-2d87ca50-b31c-4b23-ad9b-5d7aac026642.png)

That is quite huge, I think.

Reply:
This is not what I suggested. I suggested to only add version + title. Not schema + content.

Reply:
perfect 
![Capture d’écran 2022-08-22 à 14 11 58](https://user-images.githubusercontent.com/8363334/185918318-cebb2814-b4f8-4e29-a7b7-607780af9eb9.png)
 PR updated